test/parallel/test-http-chunked-304.js

```diff
@@ -12,32 +12,33 @@ var net = require('net');
 // Likewise for 304 responses. Verify that no empty chunk is sent when
 // the user explicitly sets a Transfer-Encoding header.
 
-test(204, function() {
+test(204, () => {
   test(304);
```

Comment:
I am not a big fan of nesting functions unless it is unavoidable. Now that we have `common.mustCall` to ensure that assertions will happen for sure, why don't we refactor this to sequential calls?